### DISCUSSION TITLE
Cannot find node_modules folder to create table data_source_one for example apps

### DISCUSSION THREAD

**Discussion by Sairam90** [perspective-dev/perspective#2649] [Q&A]
### Steps to Reproduce:
 1.Start python-tornado example app by running server.py

### Expected Result:
App launch

### Actual Result:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\***\\perspective\\examples\\python-tornado\\..\\..\\node_modules\\superstore-arrow\\superstore.lz4.arrow'

### Environment:
Windows

### Additional Context:
Module node_modules is abset and hence the arrow file is not found to create the table

**Comment by texodus** [comment]
This is not an adequate reproduction:

> 1.Start python-tornado example app by running server.py

The Perspective examples are built as part of a mono-repository, you generally cannot just `cd` into a directory and guess at how to build a project (which I am _guessing_ you did, hidden between the words "running `server.py`"). [Please consult the docs on how to run the example projects](https://perspective.finos.org/docs/development/#build).

```
yarn start python-tornado
```

Also, please do not open another Issue for this - the Issues list is for tracking bugs and features we intend to fix, it is not for tech support (use [Discussions](https://github.com/finos/perspective/discussions))
